src/evict/evict_lru.c

```diff
@@ -676,10 +674,8 @@ __evict_update_work(WT_SESSION_IMPL *session)
      * Scrub dirty pages and keep them in cache if we are less than half way to the clean, dirty or
      * updates triggers.
      */
-    if (bytes_inuse < (uint64_t)((target + trigger) * bytes_max) / 200) {
-        if (bytes_dirty < (uint64_t)((dirty_target + dirty_trigger) * bytes_max) / 200 &&
-          bytes_updates < (uint64_t)((updates_target + updates_trigger) * bytes_max) / 200)
-            LF_SET(WT_CACHE_EVICT_SCRUB);
+    if (bytes_inuse < (uint64_t)((target + trigger) * bytes_max) / 150) {
```

Comment:
It seems simpler to just use the trigger to decide whether to scrub dirty pages. 

I suspect the 150 here is just a number that works, but it's hard to reason about. Depending on the values of the target and trigger we could wind up stop scrubbing when the cache is below the trigger (e.g., target = 10, trigger = 90), or continue scrubbing even if the cache is past the trigger (e.g., target = 80, trigger = 90)